src/pyasm/search/upgrade/project/sthpw_upgrade.py

```diff
@@ -36,23 +41,84 @@ def upgrade_v4_5_0_a01_003(my):
 
 
 
-    def upgrade_v4_5_0_a01_002(my):
+    def upgrade_v4_5_0_a01_009(my):
         my.run_sql(''' 
         CREATE INDEX "sobject_log_stype_idx" ON sobject_log (search_type);
         ''') 
 
+    def upgrade_v4_5_0_a01_008(my):
 
+        def get_timezone_names():
+            import os
+            import tarfile
+            import dateutil.zoneinfo
 
+            zi_path = os.path.abspath(os.path.dirname(dateutil.zoneinfo.__file__))
+            zonesfile = tarfile.TarFile.open(os.path.join(zi_path, 'zoneinfo-2008e.tar.gz'))
```

Comment:
It has been tried on a windows server 2008. works like in Linux. The important thing to note is if dateutil module gets updated in the future, we may need to rename this tar.gz file